src/Compilers/CSharp/Test/Emit2/Attributes/AttributeTests_LifetimeAnnotation.cs

```diff
@@ -70,56 +63,31 @@ public void ExplicitAttribute_FromMetadata()
             comp = CreateCompilation(source, references: new[] { ref0 });
             var expected =
 @"void Program.F(ref System.Int32 i)
-    [LifetimeAnnotation(True, False)] ref System.Int32 i
+    [ScopedRef] ref System.Int32 i
 ";
             CompileAndVerify(comp, symbolValidator: module =>
             {
-                Assert.Null(GetLifetimeAnnotationType(module));
-                AssertLifetimeAnnotationAttributes(module, expected);
+                Assert.Null(GetScopedRefType(module));
+                AssertScopedRefAttributes(module, expected);
             });
         }
 
-        [Fact]
-        public void ExplicitAttribute_MissingConstructor()
```

Comment:
>ExplicitAttribute_MissingConstructor [](http://example.com/codeflow?start=20&length=36)

We should keep a missing constructor test but change the constructor to one with parameters.